code/modules/power/supermatter/supermatter.dm

```diff
@@ -152,6 +166,10 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	if(is_main_engine && GLOB.main_supermatter_engine == src)
 		GLOB.main_supermatter_engine = null
 	QDEL_NULL(soundloop)
+	distort.icon = 'icons/effects/32x32.dmi'
+	distort.icon_state = "SM_remnant"
+	distort.pixel_x = 0
+	distort.pixel_y = 0
```

Comment:
This will likely result in a hard-delete as distort still has a reference held to it on this object and will be deleted during the deletion process due to being in the contents of the SM.